Core/Src/main.c

```diff
@@ -900,11 +873,11 @@ void StartMainTask(void const * argument)
 void StartEPaper(void const * argument)
 {
   /* USER CODE BEGIN StartEPaper */
-  initEpaperTask();
+  // initEpaperTask();
```

Comment:
Remove this I think

Reply:
Did you verify it working with these functions left in, or were they commented out?